app/src/encounters/components/EncountersVessel.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import infoPanelStyles from 'styles/components/info-panel.scss';
+import iconStyles from 'styles/icons.scss';
+import ArrowLinkIcon from '-!babel-loader!svg-react-loader!assets/icons/arrow-link.svg?name=ArrowLinkIcon';
+
+
+function EncountersVessel({ vessel, userPermissions, login, openVessel }) {
+  if (vessel.info === undefined) {
```

Comment:
could we add some basic info to the end user in case of an error? nothing too fancy, but just something that's not the current endless "loading" state

Reply:
Done.